### ISSUE TITLE
DistributionNotFound: The 'tqdm==4.32.1' on OSX

### ISSUE THREAD

**Issue by AnotherWayIn** [codingo/Interlace#78]
As discussed:
Thanks

```
python3 setup.py install
running install
running bdist_egg
running egg_info
creating Interlace.egg-info
writing Interlace.egg-info/PKG-INFO
writing dependency_links to Interlace.egg-info/dependency_links.txt
writing entry points to Interlace.egg-info/entry_points.txt
writing requirements to Interlace.egg-info/requires.txt
writing top-level names to Interlace.egg-info/top_level.txt
writing manifest file 'Interlace.egg-info/SOURCES.txt'
reading manifest file 'Interlace.egg-info/SOURCES.txt'
writing manifest file 'Interlace.egg-info/SOURCES.txt'
installing library code to build/bdist.macosx-10.14-x86_64/egg
running install_lib
running build_py
creating build
creating build/lib
creating build/lib/Interlace
copying Interlace/__init__.py -> build/lib/Interlace
copying Interlace/interlace.py -> build/lib/Interlace
creating build/lib/Interlace/lib
copying Interlace/lib/threader.py -> build/lib/Interlace/lib
copying Interlace/lib/__init__.py -> build/lib/Interlace/lib
creating build/lib/Interlace/lib/core
copying Interlace/lib/core/__init__.py -> build/lib/Interlace/lib/core
copying Interlace/lib/core/__version__.py -> build/lib/Interlace/lib/core
copying Interlace/lib/core/input.py -> build/lib/Interlace/lib/core
copying Interlace/lib/core/output.py -> build/lib/Interlace/lib/core
creating build/bdist.macosx-10.14-x86_64
creating build/bdist.macosx-10.14-x86_64/egg
creating build/bdist.macosx-10.14-x86_64/egg/Interlace
copying build/lib/Interlace/__init__.py -> build/bdist.macosx-10.14-x86_64/egg/Interlace
copying build/lib/Interlace/interlace.py -> build/bdist.macosx-10.14-x86_64/egg/Interlace
creating build/bdist.macosx-10.14-x86_64/egg/Interlace/lib
creating build/bdist.macosx-10.14-x86_64/egg/Interlace/lib/core
copying build/lib/Interlace/lib/core/__init__.py -> build/bdist.macosx-10.14-x86_64/egg/Interlace/lib/core
copying build/lib/Interlace/lib/core/__version__.py -> build/bdist.macosx-10.14-x86_64/egg/Interlace/lib/core
copying build/lib/Interlace/lib/core/input.py -> build/bdist.macosx-10.14-x86_64/egg/Interlace/lib/core
copying build/lib/Interlace/lib/core/output.py -> build/bdist.macosx-10.14-x86_64/egg/Interlace/lib/core
copying build/lib/Interlace/lib/threader.py -> build/bdist.macosx-10.14-x86_64/egg/Interlace/lib
copying build/lib/Interlace/lib/__init__.py -> build/bdist.macosx-10.14-x86_64/egg/Interlace/lib
byte-compiling build/bdist.macosx-10.14-x86_64/egg/Interlace/__init__.py to __init__.cpython-37.pyc
byte-compiling build/bdist.macosx-10.14-x86_64/egg/Interlace/interlace.py to interlace.cpython-37.pyc
byte-compiling build/bdist.macosx-10.14-x86_64/egg/Interlace/lib/core/__init__.py to __init__.cpython-37.pyc
byte-compiling build/bdist.macosx-10.14-x86_64/egg/Interlace/lib/core/__version__.py to __version__.cpython-37.pyc
byte-compiling build/bdist.macosx-10.14-x86_64/egg/Interlace/lib/core/input.py to input.cpython-37.pyc
byte-compiling build/bdist.macosx-10.14-x86_64/egg/Interlace/lib/core/output.py to output.cpython-37.pyc
byte-compiling build/bdist.macosx-10.14-x86_64/egg/Interlace/lib/threader.py to threader.cpython-37.pyc
byte-compiling build/bdist.macosx-10.14-x86_64/egg/Interlace/lib/__init__.py to __init__.cpython-37.pyc
creating build/bdist.macosx-10.14-x86_64/egg/EGG-INFO
copying Interlace.egg-info/PKG-INFO -> build/bdist.macosx-10.14-x86_64/egg/EGG-INFO
copying Interlace.egg-info/SOURCES.txt -> build/bdist.macosx-10.14-x86_64/egg/EGG-INFO
copying Interlace.egg-info/dependency_links.txt -> build/bdist.macosx-10.14-x86_64/egg/EGG-INFO
copying Interlace.egg-info/entry_points.txt -> build/bdist.macosx-10.14-x86_64/egg/EGG-INFO
copying Interlace.egg-info/requires.txt -> build/bdist.macosx-10.14-x86_64/egg/EGG-INFO
copying Interlace.egg-info/top_level.txt -> build/bdist.macosx-10.14-x86_64/egg/EGG-INFO
zip_safe flag not set; analyzing archive contents...
creating dist
creating 'dist/Interlace-1.7-py3.7.egg' and adding 'build/bdist.macosx-10.14-x86_64/egg' to it
removing 'build/bdist.macosx-10.14-x86_64/egg' (and everything under it)
Processing Interlace-1.7-py3.7.egg
Removing /usr/local/lib/python3.7/site-packages/Interlace-1.7-py3.7.egg
Copying Interlace-1.7-py3.7.egg to /usr/local/lib/python3.7/site-packages
Interlace 1.7 is already the active version in easy-install.pth
Installing interlace script to /usr/local/bin

Installed /usr/local/lib/python3.7/site-packages/Interlace-1.7-py3.7.egg
Processing dependencies for Interlace==1.7
Searching for tqdm==4.32.1
Best match: tqdm 4.32.1
Processing tqdm-4.32.1-py3.7.egg
tqdm 4.32.1 is already the active version in easy-install.pth
Installing tqdm script to /usr/local/bin

Using /usr/local/lib/python3.7/site-packages/tqdm-4.32.1-py3.7.egg
Searching for netaddr==0.7.19
Best match: netaddr 0.7.19
Processing netaddr-0.7.19-py3.7.egg
netaddr 0.7.19 is already the active version in easy-install.pth
Installing netaddr script to /usr/local/bin

Using /usr/local/lib/python3.7/site-packages/netaddr-0.7.19-py3.7.egg
Searching for colorclass==2.2.0
Best match: colorclass 2.2.0
Processing colorclass-2.2.0-py3.7.egg
colorclass 2.2.0 is already the active version in easy-install.pth

Using /usr/local/lib/python3.7/site-packages/colorclass-2.2.0-py3.7.egg
Finished processing dependencies for Interlace==1.7


interlace -h

Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/2.7/bin/interlace", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3036, in <module>
    @_call_aside
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3020, in _call_aside
    f(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3049, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/pkg_resources/__init__.py", line 654, in _build_master
    ws.require(__requires__)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/pkg_resources/__init__.py", line 968, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/pkg_resources/__init__.py", line 854, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'tqdm==4.32.1' distribution was not found and is required by Interlace


pip3 list |grep tqdm
tqdm             4.32.1 
```

**Comment by xYantix** [comment]
Is there any further info on this? I've just installed the latest version of tqdm and it seems to be a higher version. Have multiple machines I can perform testing on if needed.

pip3 list | grep tqdm                
tqdm              4.36.1

**Comment by codingo** [comment]
@xYantix this should have been solved with #79, can you confirm?
